docs/overview.md

```diff
@@ -43,3 +43,18 @@ In comparison to Ethereum, Gno.land offers distinct advantages:
 2. **General-Purpose Language**: Gno.land's Gnolang is a general-purpose language, similar to Go, extending its
    usability beyond the context of blockchain. In contrast, Solidity is designed specifically for Smart Contracts on the
    Ethereum platform.
+
+## Using the Gno.land Documentation
+
+The Gno.land documentation is closely aligned with the [Diataxis](https://diataxis.fr) framework for writing software documentation. 
+Utilizing the Diataxis framework allows for organization and predictability within the docs - often, you will find 
+ideas split into:
+- their conceptual parts, providing context as to what they are & what they are used for - equating to the Diataxis' "Explanation" category,
+- their reference parts, providing concrete information on how they're implemented and can be utilized. 
+
+Apart from that:
+- How-to guides are intended to be brief but detailed technical explanations on how to execute specific tasks,
+- and tutorials must help a beginner achieve basic competence with a specific aspect of Gno.land, so that they can go on 
+  to use it for their own purposes.
+
+
```

Comment:
```suggestion
## Gno.land Documentation Overview

Gno.land's documentation adopts the [Diataxis](https://diataxis.fr/) framework, ensuring structured and predictable content. It includes:
- Conceptual explanations, offering context and usage insights.
- Detailed reference sections with implementation specifics.
- Tutorials aimed at beginners to build fundamental skills in using Gno.land.
- Concise how-to guides for specific technical tasks.
```

I like your addition, however, I think we can make it more concise, and just quote Diataxis once

Up to you

Reply:
Looks, good, I'll commit your suggestion.